css/flink.css

```diff
@@ -249,6 +249,19 @@ img.illu {margin:40px auto 60px;display:block;}
 .committer-avatar {
 	width: 50px;
 }
+
+.note {
+    border-left: 2px solid #ccc;
+    padding: 4px 0 4px 8px;
+    margin-bottom: 32px;
+	color: lightblue;
```

Comment:
This lightblue text on white background is barely readable to me. It also fails accessibility color contrast standards.

![screenshot-2021-09-06_08-10-15](https://user-images.githubusercontent.com/2392216/132168704-05a2ce9b-0189-48b7-9367-c5a929b02d83.png)